core/src/test/java/org/elasticsearch/cluster/metadata/IndexNameExpressionResolverTests.java

```diff
@@ -643,6 +644,51 @@ public void testConcreteIndicesWildcardWithNegation() {
         assertEquals(0, indexNames.length);
     }
```

Comment:
can you also add tests for the new option when resolving expressions that don't contain wildcards?